lib/utils/batch/src/index.js

```diff
@@ -121,7 +121,10 @@ const batchifyfn = (fn, delay, max, size) => {
 };
 
 // Return a function that organizes a batch of calls in groups, applies each
-// group of calls and assembles the results
+// group of calls and assembles the results.
+// Warning: In case there is an error in one of the groups the whole group fails. This behaviur
+// is due the fact that with the current implementation of transform.map we could not easily distinguish where
+// the error has happened. This logic needs to be revised in future.
```

Comment:
We need to fix `behaviur` to `behaviour` and we can replace "This logic needs to be revised in future." with github issue reference